src/Context/SQLHandler.php

```diff
@@ -244,6 +244,11 @@ public function constructSQLClause($glue, array $columns)
                 }
             }
 
+            // Check if the value is surrounded by wildcards. If so, we'll want to use a LIKE comparator.
+            if (preg_match('/^%.+%$/', $value)) {
```

Comment:
Looks good mate, just need some tests around this particular branch of code. One that builds up the SQL and one that unit tests it.